doc/api/cli.md

```diff
@@ -1052,6 +1052,15 @@ minimum allocation from the secure heap. The minimum value is `2`.
 The maximum value is the lesser of `--secure-heap` or `2147483647`.
 The value given must be a power of two.
 
+### `--test`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Starts the Node.js command line test runner. See the documentation on
+[running tests from the command line][] for more details.
```

Comment:
Are there things to note about how this interacts with other flags? Like `--inspect` and `--inspect-brk`

Reply:
I've added some code to prevent using `--test` with certain flags and called it out in the docs.